test/systems/plonk/placeholder/circuits.hpp

```diff
@@ -104,7 +104,10 @@ namespace nil {
                     public_columns_1, constant_columns_1, selector_columns_1>;
 
                 template<typename FieldType>
-                circuit_description<FieldType, placeholder_circuit_params<FieldType, arithmetization_params_1>, 4, 4> circuit_test_1() {
+                circuit_description<FieldType, placeholder_circuit_params<FieldType, arithmetization_params_1>, 4, 4> circuit_test_1(
+                    typename nil::crypto3::random::algebraic_engine<FieldType> alg_rnd = nil::crypto3::random::algebraic_engine<FieldType>(),
+                    boost::random::mt11213b rnd = boost::random::mt11213b()
```

Comment:
Can we remove rnd, we don't use it, we only use alg_rnd. We also don't use it in 2 other functions below that generate circuit descriptions.